config/routes.rb

```diff
@@ -61,6 +61,9 @@
     get "/dashboard", to: "dashboard#index"
     get "/orders/:order_id", to: "orders#show"
     get "/items", to: "items#index"
+    match "/items/:item_id/toggle_active", :to => "items#toggle_active", :as => 'merchant_item_active', :via => :patch
+
```

Comment:
I think we can get a ReSTful implementation by:

```ruby
namespace :merchant do
 # ...
  patch "/items/:item_id", :to => "items#update"
end
```

Reply:
Regarding `:as => 'merchant_item_active'`, are you using that path helper prefix anywhere?

Reply:
I can the path helper. I tried using it, but it wasn't working for me, so I went back to the URL.

Reply:
It will still have to go through a non-RESTful route, since update is being saved for later. Or, we could route it through another controller all together.